runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -386,10 +403,27 @@ private void calculate()
 			int actionCount = 0;
 			int neededXP = targetXP - currentXP;
 			SkillAction action = slot.getAction();
+
+			SkillBonus[] excludedBonuses = action.getExcludedSkillBonuses();
+			if (excludedBonuses.length > 0)
+			{
+				if (Arrays.stream(excludedBonuses).anyMatch(excludedBonus -> currentBonus.stream().anyMatch(currentBonus -> excludedBonus == currentBonus)))
+				{
+					slot.setText("N/A for current bonuses");
```

Comment:
I think instead we would want something more subtle. If a player selects Zealot robes and looks at scattering ashes for instance (I understand zealot robes will be added in a separate PR, but that is probably the main reason we want this enhancement) we should just show the normal text and xp value. Maybe we could add an icon and/or tooltip to indicate some subset of selected bonuses will not apply to it.

Reply:
Yeah I was mostly trying to make it obvious that this action doesn't get any multiplier when using this bonus. I'll look into the icon / tooltip suggestion.